File: src/main/java/ru/ioleksiv/telegram/bot/core/api/model/objects/inline/queryresult/FormattedInlineResult.java
package ru.ioleksiv.telegram.bot.core.api.model.objects.inline.queryresult;

import com.fasterxml.jackson.annotation.JsonProperty;

public abstract class FormattedInlineResult extends CaptionInlineResult {
    /**
     * parse_mode	String	Optional. Send Markdown or HTML, if you want Telegram apps to show bold, italic, fixed-width text or inline URLs in the media caption.
     */
    @JsonProperty("parse_mode")
    private String parseMode = null;

    FormattedInlineResult(String type) {
        super(type);
    }

    public final String getParseMode() {
        return parseMode;
    }

    public final void setParseMode(String parseMode) {
        this.parseMode = parseMode;
    }

}

File: server-api/src/main/kotlin/io/github/oleksivio/tl/kbot/server/api/objects/inline/queryresult/FormattedInlineResult.kt
package io.github.oleksivio.tl.kbot.server.api.objects.inline.queryresult

import com.fasterxml.jackson.annotation.JsonProperty

abstract class FormattedInlineResult : CaptionInlineResult() {
    /**
     * parse_mode [String] Optional. Send Markdown or HTML, if you want Telegram apps to show bold, italic, fixed-width text
     * or inline URLs in the media caption.
     */
    @JsonProperty("parse_mode")
    var parseMode: String? = null
}
